docs/HowTo/Get-Started/Use-Docker.md

```diff
@@ -28,6 +28,14 @@ Display the EthSigner command line help using the Docker image:
 !!! note
     `latest` runs the latest cached version. To pull the latest version, use `docker pull pegasyseng/ethsigner:latest`.
 
+You can specify [environment variables] with the docker image instead of the command line options.
+
+=== "example"
+
+    ```bash
+    docker run --mount type=bind,source=/home/me/me_node,target=/var/lib/ethsigner -e ETHSIGNER_CONFIG_FILE=/var/lib/ethsigner/config.toml pegasyseng/ethsigner:latest file-based-signer
+    ```
+
```

Comment:
that looks good. 

nit-pick ... this particular example uses config file option from environment variable, does it make sense to have another alternate example to use environment variables without config file? i.e. 

`docker run -p <listenPort>:8545 -v <~/myKeyFile>:/opt/ethsigner/keyfile -v <~/myPasswordFile>:/opt/ethsigner/passwordfile pegasyseng/ethsigner:latest --chain-id=2018 --downstream-http-host=<PantheonHost> --downstream-http-port=8590 --http-listen-host=0.0.0.0 file-based-signer -k /opt/ethsigner/keyfile -p /opt/ethsigner/passwordfile`

can also be written as

`docker run -p <listenPort>:8545 -v <~/myKeyFile>:/opt/ethsigner/keyfile -v <~/myPasswordFile>:/opt/ethsigner/passwordfile -e ETHSIGNER_CHAIN_id=2018 -e ETHSIGNER_DOWNSTREAM_HTTP_HOST=<BesuHost> -e ETHSIGNER_DOWNSTREAM_HTTP_PORT=8590 -e ETHSIGNER_HTTP_LISTEN_HOST=0.0.0.0 -e ETHSIGNER_FILE_BASED_SIGNER_KEY_FILE=/opt/ethsigner/keyfile -e  ETHSIGNER_FILE_BASED_SIGNER_PASSWORD_FILE=/opt/ethsigner/passwordfile pegasyseng/ethsigner:latest file-based-signer `